src/main/java/ecostruxure/rate/calculator/dal/db/ProfileDAO.java

```diff
@@ -252,12 +255,13 @@ public Profile get(TransactionContext context, UUID id) throws Exception {
 
     private Profile createProfile(Connection connection, Profile profile) throws Exception {
 
-        try (PreparedStatement stmt = connection.prepareStatement("INSERT INTO dbo.Profiles (profile_id, name, currency, country_id, resource_type, annual_cost, effectiveness, annual_hours, effective_work_hours, hours_per_day, is_archived) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)", Statement.RETURN_GENERATED_KEYS)) {
+        try (PreparedStatement stmt = connection.prepareStatement("INSERT INTO dbo.Profiles (profile_id, name, currency, geography_id" +
```

Comment:
Lav venligst en string var med query i, i stedet for at skrive det direkte i parameter på prepareStatement(). Tjek eventuelt linje 81-87 som eksempel på hvordan du laver en variabel